packages/www/docs/api-reference/vod/list.mdx

```diff
@@ -105,29 +105,13 @@ axios
     "createdAt": 1644546528636,
     "videoSpec": {
         "format": "mp4",
```

Comment:
Should probably revert these changes to docs so you're able to rebase (tons of conflicts).

Reply:
Reverted all the docs changes!